src/request_builder.rs

```diff
@@ -120,7 +120,10 @@ impl RequestBuilder {
         self
     }
 
-    /// Sets the body of the request.
+    /// Pass an `AsyncRead` stream as the request body.
+    /// # Mime
+    ///
+    /// The encoding is set to `application/octet-stream`.
```

Comment:
```suggestion
    /// The `content-type` is set to `application/octet-stream`.
```

& similar spots

(To not confuse with other "encoding" headers.)